core/src/main/resources/error/error-classes.json

```
@@ -581,6 +581,12 @@
     ],
     "sqlState" : "42805"
   },
+  "IDENTIFIER_TOO_MANY_NAME_PARTS" : {
+    "message" : [
+      "<quoted> is not a valid <identifier> as it has more than 2 name parts."
```

Let's don't pass TableIdentifier or FunctionIdentifier, and just say:
```suggestion
      "<quoted> is not a valid identifier as it has more than 2 name parts."
```
sine the camel-case ids might confuse users and are not so much useful.

---

Done

---

This keeps confusing me. Is it only called for tables? Isn't schema.table.c1 a perfectly fine identifier (for a column)?
(and we haven't even talked about (nested) struct fields...)

---

@srielau sorry for the late response, I missed this comment... Yep, `schema.table.c1` is a valid identifier for column name and for struct field name.